components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/gateway/v1/impl/RedeployApiApiServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package org.wso2.carbon.apimgt.rest.api.gateway.v1.impl;
+
+import org.json.JSONObject;
+import org.wso2.carbon.apimgt.gateway.InMemoryAPIDeployer;
+import org.wso2.carbon.apimgt.impl.APIConstants;
+import org.wso2.carbon.apimgt.rest.api.gateway.v1.*;
+import org.wso2.carbon.apimgt.rest.api.gateway.v1.dto.*;
+
+import org.apache.cxf.jaxrs.ext.multipart.Attachment;
+import org.apache.cxf.jaxrs.ext.MessageContext;
+
+import org.wso2.carbon.apimgt.rest.api.gateway.v1.dto.DeployResponseDTO;
+import org.wso2.carbon.apimgt.rest.api.gateway.v1.dto.ErrorDTO;
+
+import java.util.List;
+
+import java.io.InputStream;
+import java.util.Map;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+
```

Comment:
Please remove the unnecessary new lines.

Reply:
https://github.com/wso2/carbon-apimgt/pull/9546/commits/dfbd644d35add8643bb9aa652caba4e12f511b5b